src/clr-icons/shapes/technology-shapes.ts

```diff
@@ -1250,6 +1280,12 @@ export const TechnologyShapes: any = {
   inductor: ClrShapeInductor,
   resistor: ClrShapeResistor,
   'host-group': ClrShapeHostGroup,
+  'storage-adapter': ClrShapeStorageAdapter,
+  'node-cluster-group': ClrShapeNodeClusterGroup,
```

Comment:
After some thinking, I think it's better to go with even shorter and more generic and more common names. I think it's better to make the following updates: `node-cluster-group` would become `node-groups` and `node-cluster` would be just `nodes`.